LICENSE

```diff
@@ -1,25 +1,21 @@
-Copyright (c) 2013 The github.com/redis/go-redis Authors.
-All rights reserved.
+MIT License
 
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are
-met:
+Copyright (c) 2022-2023, Redis, inc.
```

Comment:
Really?

Reply:
@uglide ? I was just lining things up... Someone help me - I am opinion free here - this is just a copy/paste from redis-py

Reply:
@chayim we shouldn't change the copyright. The aim is to align all clients under Redis org to the same MIT license. 

@vmihailenco you are the copyright owner for most of the code. Is it something that you can consider? Both licenses are permissive, true-OSS.

Reply:
As far as I am concerned the MIT license is fine, but the copyright should stay the same.

"The Authors" means the people who contributed the code (which includes Redis, inc.). Maintainers change with time and don't have anything to do with it.